setup/native-script.ps1

```diff
@@ -34,6 +37,12 @@ if (!$env:JAVA_HOME) {
 
 # install NativeScript CLI
 write-host -BackgroundColor Black -ForegroundColor Yellow "Installing NativeScript CLI"
-npm install -g nativescript
+
+$oldPathUser = [Environment]::GetEnvironmentVariable("PATH", "User")
+$pathMachine = [Environment]::GetEnvironmentVariable("PATH", "Machine")
+$myPath = [Environment]::GetEnvironmentVariable("PATH")
+[Environment]::SetEnvironmentVariable("PATH", "$myPath;$oldPathUser;$pathMachine;$env:ProgramFiles\nodejs")
```

Comment:
According to MSDN the value will not be set until PowerShell is restarted:
```
Or at least it didn’t show up until we restarted PowerShell. (Or started up a new instance of PowerShell.)
```
Maybe we have to use full path to npm (from node installation). 
[source](https://technet.microsoft.com/en-us/library/ff730964)